src/resources/views-elfinder/elfinder.blade.php

```diff
@@ -1,25 +1,25 @@
 @extends('backpack::layout')
 
 @section('after_scripts')
-    <!-- jQuery and jQuery UI (REQUIRED) -->
+    {{-- jQuery and jQuery UI (REQUIRED) --}}
     <link rel="stylesheet" href="//ajax.googleapis.com/ajax/libs/jqueryui/1.10.4/themes/smoothness/jquery-ui.css" />
-    <!-- <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script> -->
+    {{-- <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script> --}}
     <script src="//ajax.googleapis.com/ajax/libs/jqueryui/1.10.4/jquery-ui.min.js"></script>
 
-    <!-- elFinder CSS (REQUIRED) -->
+    {{-- elFinder CSS (REQUIRED) --}}
```

Comment:
That's either a redundant comment OR it should remain (it's a comment about the HTML page, not about the logic on the page).